### DISCUSSION TITLE
segment

### DISCUSSION THREAD

**Discussion by twilio-jyoung** [twilio-labs/paste#2036] [Request: Icon] [answered]
segment icon please!

**Comment by loreina** [comment]
Hi @twilio-jyoung! Could you share your use case for the icon and a screenshot of where the icon would be used in context? Also, would you happen to have an svg of the logo? Thanks!

**Comment by loreina** [comment, marked as answer]
We added the icon in #2059 and it should be added to Paste in our next release. 😊
